.github/workflows/build_and_test.yml

```diff
@@ -273,6 +273,44 @@ jobs:
         cd docs
         jekyll build
 
+  webui-tests-with-chrome:
+    name: WebUI tests with chrome
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+    - name: Cache Maven local repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: webui-tests-with-chrome-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          webui-tests-with-chrome-maven-
+    - name: Install Java 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Install Chrome and ChromeDriver
+      run: |
+        sudo apt update
+        sudo apt install google-chrome-stable
+        sudo apt install chromium-chromedriver
+    - name: Run WebUI tests with Maven
+      run: |
+        export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
+        export MAVEN_CLI_OPTS="--no-transfer-progress"
+        mkdir -p ~/.m2
+        ./build/mvn -Dspark.test.webdriver.chrome.driver=/usr/bin/chromedriver \
+          -Dguava.version=25.0-jre -Djava.version=11 -Dtest.default.exclude.tags=  -Dtest=none \
```

Comment:
@sarutak . Apache Spark 3.1's Guava version is still `<guava.version>14.0.1</guava.version>`, isn't it?
Why do you pig-back something like this important thing silently?

Reply:
Yes. I know the official version of Guava is `14.0.1`. Actually, I noticed the current ChromeDriver can't use `guava 14.0.1` so I intended to use `25.0` only for `ChromeUISeleniumSuite` and `ChromeUIHistoryServerSuite`.
Do you have any better idea rather than use `25.0`?

Reply:
If the test suite is broken with the default Apache Spark distribution, we need to revert the root cause which broke the test suite. IIRC, this test suite is tested with the default configuration initially.
> Actually, I noticed the current ChromeDriver and RemoteWebDriver can't use guava 14.0.1 

Do we have a JIRA issue for that test suite failure?

Reply:
The first time those tests are added, they works with `guava 14.0.1`.
I believe #28585 (SPARK-31765) requires upgraded RemoteWebDriver and it requires `25.0-jre`.
That PR is for a small security issue.

> Do we have a JIRA issue for that test suite failure?

Yes. But we need to modify the title.
https://issues.apache.org/jira/browse/SPARK-31996

Reply:
Thank you for the pointer, @sarutak . BTW, there are many request for Guava update. I believe we can handle this in the same group.

Reply:
> BTW, there are many request for Guava update.

Yeah, I know. So, should we revert that PR? It's a security fix but for the test code, or waiting for Guava updated?

Reply:
What about just not adding this test? :)
How important is it? does this cover new ground?

Reply:
It can be one option but I'm wondering if we can make much better coverage for UI tests somehow.
I often see that UI tests are done by "manual" especially when the change is related to JavaScript.
`UISeleniumSuite`, `HistoryServerSuite` and other similar suites are enough for the static part of the UI like but not enough for the dynamic part which dynamically changes DOM or CSS by JavaScript.
Those existing suites use `HtmlUnitDriver` but its JavaScript support is not enough.
So I think it's better if we can run test with real browsers in the CI.
`ChromeDriver` is similar to `HtmlUnitDriver` but it uses the real Chrome so we can test JavaScript related stuffs.
`ChromeUISeleniumSute` and `ChromeUIHistoryServerSuite` use `ChromeDriver` rather than `HtmlUnitDriver` and all tests in the suites are what need JavaScript support so I think they cover new ground.